lib/VM/JSLib/GlobalObject.cpp

```diff
@@ -764,6 +777,11 @@ void initGlobalObject(Runtime *runtime, const JSLibFlags &jsLibFlags) {
         intl::createIntlObject(runtime)));
   }
 #endif
+
+#ifdef HERMES_ENABLE_FUZZILLI
```

Comment:
Instead of defining these functions inside of Hermes internally, I recommend instead defining them in fuzzili.cpp.
That should remove the need for adding the `HERMES_ENABLE_FUZZILLI` preprocessor variable and changing all the internal string and function files.